_posts/2020-08-05-#19160.md

```diff
@@ -7,17 +7,47 @@ authors: [ryanofsky]
 components: ["build system"]
 host: ryanofsky
 status: upcoming
-commit:
+commit: 0602f87
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+These review club notes are for PR [#19160 multiprocess: Add basic spawn and IPC support](https://github.com/bitcoin/bitcoin/pull/19160) ([branch](https://github.com/ryanofsky/bitcoin/commits/pr/ipc-echo.7)), focused on the second eponymous commit: [`aa4d626`](https://github.com/ryanofsky/bitcoin/commit/aa4d626db1628d5e29572eccfd7fa2d99edeeb60).
+
```

Comment:
Maybe add a sentence to define IPC, with a link (https://en.wikipedia.org/wiki/Inter-process_communication?)